coverage/lcov-report/core/AuthorizationCodeGrant.ts.html

```diff
@@ -0,0 +1,1003 @@
+
```

Comment:
Did you check in the coverage reports deliberately? I usually add them to gitignore